google/data_source_google_redis_instance_test.go

```diff
@@ -0,0 +1,39 @@
+package google
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+)
+
+func TestAccRedisInstanceDatasource_basic(t *testing.T) {
+	t.Parallel()
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:  func() { testAccPreCheck(t) },
+		Providers: testAccProviders,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccRedisInstanceDatasourceConfig(randString(t, 10)),
+				Check: resource.ComposeTestCheckFunc(
+					checkDataSourceStateMatchesResourceState("data.google_redis_instance.redis", "google_redis_instance.redis"),
+				),
+			},
+		},
+	})
+}
+
+func testAccRedisInstanceDatasourceConfig(suffix string) string {
+	return fmt.Sprintf(`
+resource "google_redis_instance" "redis" {
+	name               = "redis-test-%s"
```

Comment:
tabs v spaces in this config, if you could just make them consistent one way or the other